boards/riscv32/m2gl025_miv/m2gl025_miv_defconfig

```diff
@@ -0,0 +1,17 @@
+CONFIG_RISCV32=y
+CONFIG_SOC_SERIES_RISCV32_MIV=y
+CONFIG_SOC_RISCV32_MIV=y
+CONFIG_BOARD_M2GL025=y
+CONFIG_CONSOLE=y
+CONFIG_SERIAL=y
+CONFIG_UART_MIV=y
+CONFIG_UART_MIV_PORT_0=y
+CONFIG_UART_MIV_PORT_0_BAUD_RATE=115200
+CONFIG_UART_MIV_PORT_0_NAME="uart0"
+CONFIG_UART_CONSOLE=y
+CONFIG_UART_CONSOLE_ON_DEV_NAME="uart0"
+CONFIG_PLIC=y
+CONFIG_RISCV_MACHINE_TIMER=y
+CONFIG_GPIO=n
+CONFIG_BOOT_BANNER=y
```

Comment:
don't need BOOT_BANNER in the defconfig

Reply:
can you remove the boot banner flag from the defconfig?

Reply:
it is now removed.